apiserver/facades/controller/migrationmaster/facade.go

```diff
@@ -212,6 +244,15 @@ func (api *API) Export() (params.SerializedModel, error) {
 	return serialized, nil
 }
 
+// ProcessRelations is masked on older versions of the migration master API
+func (api *APIV1) ProcessRelations(_, _ struct{}) {}
+
+// ProcessRelations processes any relations that need updating after an export.
+// This should help fix any remoteApplications that have been migrated.
+func (api *API) ProcessRelations(args params.ProcessReleations) error {
+	return nil
```

Comment:
Future PRs will address this, for now this PR is mainly aimed at getting the SAAS block moved.